docs/intro/intro_scaling.rst

```diff
@@ -151,15 +157,15 @@ This results in some time savings.
     In [1]: %timeit bin_packing(weights)
     294 ms ± 9.39 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)
 
-Construct the models individually
-=================================
+Model Construction
```

Comment:
Same comment, I would like the section titles to indicate what specifically the advice in each section is